get-thin-edge_io.sh

```diff
@@ -134,8 +134,8 @@ esac
 rm -R /tmp/tedge
 
 # Test if tedge command is there and working
-tedge help >/dev/null
-if [ $? -eq 0 ]; then
+if ! tedge help >/dev/null;
+then
```

Comment:
I am pretty sure that these are not equal change. Actually, after I ran the script in this PR, it went to the `else` case, even though the original version went to `then` case. We have to fix here.

Reply:
Very good catch, thanks!